### ISSUE TITLE
NTR: Vasoactive intestinal peptide (VIP) high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8257]
New term request Vasoactive intestinal peptide (VIP) high in blood (elthat_100311055115):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000564
- label: Elevated circulating vasoactive intestinal peptide concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: PMID:31559013
- def: The concentration of vasoactive intestinal peptide (VIP) in the blood circulation is above the upper limit of normal.
- comment: VIP is a gut peptide hormone originally reported as a vasodilator in 1970, has multiple physiological and pathological effects on development, growth, and the control of neuronal, epithelial, and endocrine cell functions that in turn regulate ion secretion, nutrient absorption, gut motility, glycemic control, carcinogenesis, immune responses, and circadian rhythms.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8257
- synonyms:
